tab_automl/automl/datasets.py

```diff
@@ -14,7 +16,18 @@ class ClassificationDataset:
 
     def __init__(self, path):
         # reading the dataset from source
-        self.data = pd.read_csv(path)
+        if path.endswith(".txt"):
+            self.data = pd.read_table(path, delimiter='\s')
+        elif path.endswith(".json"):
+            self.data = pd.read_json(path)
+        elif path.endswith(".xlsx"):
+            self.data = pd.read_excel(path)
+        elif path.endswith(".sqlite"):
+            table_name = input("table name :")
+            db = sqlite3.connect(path)
+            self.data = pd.read_sql_query(f'Select * from {table_name}', db)
+        else:
```

Comment:
Use elif for `csv` too.

Reply:
done

Reply:
Hey @Tihsrah , it is okay. Can you do this for the rest data classes? Otherwise it's perfect.

Reply:
yes sir done (sorry i forgot to update that last time)